types/react-native-charts-wrapper/index.d.ts

```diff
@@ -36,9 +36,9 @@ export type EasingType =
     | 'EaseOutBounce'
     | 'EaseInOutBounce';
 /**
- * use `processColor` from `react-native` to generate the corrsponding number from a color (named, hex, rgba, etc.).
+ * use `processColor` from `react-native` to generate the corresponding number from a color (named, hex, rgba, etc.).
  */
-export type Color = number;
+export type Color = ProcessedColorValue | null | undefined;
```

Comment:
Do you really need to use `undefined` here? I can see that every time it is used, it is something like `gridBackgroundColor?: Color;`

Reply:
It's just becase the return type of `processColor` is `ProcessedColorValue | null | undefined` so now `Color` matches.

Also, the tests fail due to this line without `undefined` https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-native-charts-wrapper/react-native-charts-wrapper-tests.tsx#L402

Reply:
Perhaps it would be better to not explicitly define the type here, but rather get it from `processColor`?

E.g.

```suggestion
export type Color = ReturnType<typeof processColor>;
```

(Assuming `processColor` is imported in this module, otherwise that needs to be done too.)

Reply:
Added that while I was making edits anyways.